svgdigitizer/entrypoint.py

```diff
@@ -313,7 +314,7 @@ def _create_package(metadata, csvname, outdir):
             | package_schema.get_field(name).to_dict()
         )
 
-    package.resources[0].schema.metadata_defaults["fields"] = new_fields
+    package.resources[0].schema = Schema.from_descriptor({'fields': new_fields})
```

Comment:
I will not add news for this change since the bug was not part of the latest version.